drivers/gles3/rasterizer_scene_gles3.h

```diff
@@ -157,9 +157,10 @@ class RasterizerSceneGLES3 : public RasterizerScene {
 			uint32_t view_index;
 
 			// make sure this struct is padded to be a multiple of 16 bytes for webgl
-			float pad[1];
+			float pad[3];
 
 		} ubo_data;
+		static_assert(sizeof(SceneDataUBO) % 16 == 0);
```

Comment:
Ah some compilers fail with:
```
 In file included from platform/javascript/os_javascript.cpp:35:
In file included from ./drivers/gles3/rasterizer_gles3.h:35:
./drivers/gles3/rasterizer_scene_gles3.h:163:47: error: 'static_assert' with no message is a C++17 extension [-Werror,-Wc++17-extensions]
                static_assert(sizeof(SceneDataUBO) % 16 == 0);
                                                            ^
                                                            , ""
1 error generated.
```

And indeed we compile with C++14 the `3.x` branch (C++17 for `master`). You can add a message like this:
```
static_assert(sizeof(SceneDataUBO) % 16 == 0, "SceneDataUBO size must be a multiple of 16 bytes");
```

Please make the change by amending the commit (`git commit --amend` and then `git push --force` to update the PR).